src/lib/utils/array.ts

```diff
@@ -0,0 +1,6 @@
+export function shuffleArray(array: Array<any>): Array<any> {
+  return array
+    .map((value) => ({ value, sort: Math.random() }))
```

Comment:
Pinaillage absolu, désolé ;-)

```suggestion
    .map((value) => ({ value, sortWeight: Math.random() }))
```

('rank' ou 'sortOrder' seraient plus clairs, mais ils laisseraient entendre que c'est un entier)

Reply:
Why not 🤷‍♂️